tests/Fixer/PhpTag/NoShortEchoTagFixerTest.php

```diff
@@ -31,7 +31,7 @@ final class NoShortEchoTagFixerTest extends AbstractFixerTestCase
      */
     public function testFix($expected, $input = null)
     {
-        if (!ini_get('short_open_tag')) {
+        if (\PHP_VERSION_ID < 50400 && !ini_get('short_open_tag')) {
```

Comment:
We don't support PHP version lower than `5.4`, so this change is pointless.

Reply:
Done: c3535e6f59568504b9a81418418a2194af780bcc

Reply:
The `if` can be completely removed here too.

Reply:
The `if` can be completely removed here too.

Reply:
Done: 2721a030d64f2f00ef3952a5077857b545609a25